### ISSUE TITLE
getSectionText() gives StringIndexOutOfBoundsException

### ISSUE THREAD

**Issue by GoogleCodeExporter** [MER-C/wiki-java#21]
```
I want to do:

getSectionText("User:Me/somepage", 0);

Content of "User:Me/somepage" is 

  ==Section==
  text
  ==Section
  test

what i got:

Exception in thread "Thread-7" java.lang.StringIndexOutOfBoundsException: 
String index out of range: -26
    at java.lang.String.substring(String.java:1958)
    at org.wikipedia.Wiki.getSectionText(Wiki.java:1590)
    at cuploader.frames.FUpload$3.run(FUpload.java:429)
    at java.lang.Thread.run(Thread.java:722)

In other words: if section 0 contains nothing, all hell breaks loose.

Cheers, Yarl
```

Original issue reported on code.google.com by `carebone` on 1 Oct 2012 at 11:51
